sdcm/nemesis.py

```diff
@@ -537,6 +537,8 @@ def _terminate_cluster_node(self, node):
         self.monitoring_set.reconfigure_scylla_monitoring()
 
     def disrupt_nodetool_decommission(self, add_node=True, disruption_name=None):
+        if self._is_it_on_kubernetes() and disruption_name is None:
+            self.target_node = self.cluster.nodes[-1]
```

Comment:
Is it be better to use `self.set_last_node_as_target()`?

Reply:
nice advise, swtiched to it